### PR TITLE
New indicator layouts fixed

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
New layouts to indicators

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/Layouts/layout-details-Vulnerability.json [hunk 1]
```diff
@@ -1,457 +1,428 @@
 {
- "typeId": "Vulnerability",
```
Comment (altmannyarden) [inline Packs/CommonTypes/Layouts/layout-details-Vulnerability.json:2]: lets rollback this changes to the layout

File: Packs/CommonTypes/Layouts/layout-edit-Vulnerability.json [hunk 2]
```diff
@@ -1,198 +1,199 @@
 {
- "typeId": "Vulnerability",
- "kind": "edit",
- "layout": {
```
Comment (altmannyarden) [inline Packs/CommonTypes/Layouts/layout-edit-Vulnerability.json:4]: let's rollback those changes

File: Packs/CommonTypes/Layouts/layout-indicatorsDetails-STIX_Attack_Pattern.json [hunk 3]
```diff
@@ -1,415 +1,416 @@
...
+    "TypeName": "STIX Attack Pattern",
     "kind": "indicatorsDetails",
     "layout": {
-		"TypeName": "",
+        "TypeName": "",
         "id": "STIX Attack Pattern",
         "kind": "indicatorsDetails",
         "name": "",
         "system": false,
-		"tabs": [
```
Comment (altmannyarden) [inline Packs/CommonTypes/Layouts/layout-indicatorsDetails-STIX_Attack_Pattern.json:10]: let's rollback those changes

**Review by altmannyarden — CHANGES_REQUESTED** [review]
let's rollback all changes to the old layout files

File: Packs/CommonTypes/ReleaseNotes/3_2_47.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+
+#### Layouts
+- **domainRepUnified**
+- **STIX Attack Pattern**
+- **hostRep**
+- **STIX Report**
+##### Intrusion Set
```
Comment (altmannyarden) [inline Packs/CommonTypes/ReleaseNotes/3_2_47.md:7]: Should fix the release notes with all those **<>**

File: Packs/CommonTypes/Layouts/layoutscontainer-Campaign.json [hunk 5]
```diff
@@ -517,13 +442,13 @@
                         ],
                         "maxW": 3,
                         "minH": 1,
-                        "minW": 1,
                         "moved": false,
                         "name": "Campaign Description",
```
Comment (altmannyarden) [inline Packs/CommonTypes/Layouts/layoutscontainer-Campaign.json:446]: Should it be "Campaign Description" or the field "Description"?

**Review by altmannyarden — APPROVED** [review]

File: Packs/CommonTypes/ReleaseNotes/3_2_47.md (hunk 4 above)
Comment (Ni-Knight) [reply Packs/CommonTypes/ReleaseNotes/3_2_47.md:20]: Done

**Pull request merged**